src/snowflake/snowpark/_internal/server_connection.py

```diff
@@ -290,7 +291,7 @@ def upload_stream(
         # https://docs.python.org/3/library/io.html#io.IOBase.close
         except ValueError as ex:
             if input_stream.closed:
-                raise SnowparkClientExceptionMessages.SERVER_UDF_UPLOAD_FILE_STREAM_CLOSED(
+                raise SnowparkClientExceptionMessages.SERVER_UPLOAD_FILE_STREAM_CLOSED(
```

Comment:
Is there a way to check weather this function was called by udf or not?

Reply:
we can add an`is_in_udf` argument to the internal upload_stream method, and set it accordingly based on where it's called.